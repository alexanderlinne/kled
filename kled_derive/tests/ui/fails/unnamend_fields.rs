#[macro_use]
extern crate kled_derive;

#[operator(type = "flow")]
pub struct Map<UnaryOp>(UnaryOp);

fn main() {}
