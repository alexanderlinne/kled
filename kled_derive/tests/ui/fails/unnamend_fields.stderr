error: `#[operator]` deriving from structs with unnamed fields is not yet implemented
 --> $DIR/unnamend_fields.rs:5:24
  |
5 | pub struct Map<UnaryOp>(UnaryOp);
  |                        ^^^^^^^^^
